test/functional/p2p_compactblocks.py

```diff
@@ -561,6 +561,15 @@ def test_getblocktxn_handler(self, test_node):
             assert_equal(test_node.last_message["block"].block.sha256, int(block_hash, 16))
             assert "blocktxn" not in test_node.last_message
 
+        # Request with out-of-bounds tx index results in disconnect
+        bad_peer = self.nodes[0].add_p2p_connection(TestP2PConn())
+        block_hash = node.getblockhash(chain_height)
+        block = from_hex(CBlock(), node.getblock(block_hash, False))
```

Comment:
25370: this should fix linter and tests
```suggestion
        bad_peer = self.nodes[0].add_p2p_connection(TestP2PConn(cmpct_version=1))
        block_hash = node.getblockhash(chain_height)
        block = FromHex(CBlock(), node.getblock(block_hash, False))
```